zappa/core.py

```diff
@@ -309,15 +309,17 @@ def __init__(
             self.manylinux_suffix_start = "cp36m"
         elif self.runtime == "python3.7":
             self.manylinux_suffix_start = "cp37m"
-        else:
+        elif self.runtime == "python3.8":
             # The 'm' has been dropped in python 3.8+ since builds with and without pymalloc are ABI compatible
             # See https://github.com/pypa/manylinux for a more detailed explanation
             self.manylinux_suffix_start = "cp38"
+        else:
+            self.manylinux_suffix_start = "cp39"
 
         # AWS Lambda supports manylinux1/2010 and manylinux2014
         manylinux_suffixes = ("2014", "2010", "1")
         self.manylinux_wheel_file_match = re.compile(
-            f'^.*{self.manylinux_suffix_start}-manylinux({"|".join(manylinux_suffixes)})_x86_64.whl$'
+            f'^.*{self.manylinux_suffix_start}-(manylinux_\d+_\d+_x86_64[.])?manylinux({"|".join(manylinux_suffixes)})_x86_64[.]whl$'
```

Comment:
I'm probably missing the basics here, sorry for the basic questions.
For my own learning, two small points:
1. Why is the `(manylinux_\d+_\d+_x86_64[.])?` needed?
2. Why the `[.]` addition

Reply:
For 1. it seems the file name format has been changed for the newer manywheel packages. As for 2. the [] escapes the “.” so that only a “.” matches.

Reply:
got it! 
1. 👍🏼 
2. Nice catch, basically the regex wasn't 100% correct before. I'm used to `\.` to match a regular "." that's why I was confused